include/alpaka/dev/cpu/SysInfo.hpp

```diff
@@ -39,223 +39,203 @@
 #include <stdexcept>
 #include <string>
 
-namespace alpaka
+#if BOOST_ARCH_X86
+#    if BOOST_COMP_GNUC || BOOST_COMP_CLANG || BOOST_COMP_PGI
+#        include <cpuid.h>
+#    elif BOOST_COMP_MSVC || defined(BOOST_COMP_MSVC_EMULATED)
+#        include <intrin.h>
+#    endif
+#endif
+
+namespace alpaka::cpu::detail
 {
-    namespace cpu
-    {
-        namespace detail
-        {
-            constexpr int NO_CPUID = 0;
-            constexpr int UNKNOWN_CPU = 0;
-            constexpr int UNKNOWN_COMPILER = 1;
+    constexpr int NO_CPUID = 0;
+    constexpr int UNKNOWN_CPU = 0;
+    constexpr int UNKNOWN_COMPILER = 1;
 #if BOOST_ARCH_X86
 #    if BOOST_COMP_GNUC || BOOST_COMP_CLANG || (!BOOST_COMP_MSVC_EMULATED && defined(__INTEL_COMPILER))               \
         || BOOST_COMP_PGI
```

Comment:
```suggestion
#    if BOOST_COMP_GNUC || BOOST_COMP_CLANG || BOOST_COMP_PGI
```
Could you also remove them here please? Sorry for not noticing earlier.

Reply:
No problem, done.